docs/topics/ref_shared-messaging-work-queue-resources.adoc

```diff
@@ -0,0 +1,10 @@
+[id='shared-messaging-work-queue-resources_{context}']
+= Shared Messaging Work Queue Resources
+
+To learn more about messaging patterns and Red{nbsp}Hat JBoss A-MQ for OpenShift, follow the links below:
```

Comment:
I know it used to be A-MQ, but it's AMQ now.

Reply:
Also not sure if it's still called "Red Hat JBoss AMQ" or just "Red Hat AMQ"; the [docs](https://access.redhat.com/documentation/en-us/red_hat_amq/7.0/) are not particularly clear on that either. I think it's probably the latter, but at least we should be consistent.

Reply:
As of 7, it's AMQ (no hyphen) everywhere.

Reply:
I think we should probably use the new terminology (= AMQ) even if we temporarily use A-MQ 6.

Reply:
Agreed, I've chnaged _A-MQ_ to _AMQ_ everywhere in the text. Thank you.